extensions/cdc-mysql/src/main/java/com/hazelcast/jet/cdc/mysql/MySqlCdcSources.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2020 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.cdc.mysql;
+
+import com.hazelcast.jet.annotation.EvolvingApi;
+import com.hazelcast.jet.cdc.ChangeEvent;
+import com.hazelcast.jet.cdc.impl.AbstractSourceBuilder;
+import com.hazelcast.jet.cdc.impl.ChangeEventJsonImpl;
+import com.hazelcast.jet.cdc.impl.PropertyRules;
+import com.hazelcast.jet.pipeline.StreamSource;
+import org.apache.kafka.connect.data.Values;
+
+/**
+ * Contains factory methods for creating change data capture sources
+ * based on MySQL databases.
+ *
+ * @since 4.1
+ */
+@EvolvingApi
+public final class MySqlCdcSources {
```

Comment:
`MySql` -> `Mysql`, even though I'm not sure, JDBC connector uses this way internally.

Reply:
Seems harder to read to me... `MySQL` would be the perfectly correct version, but that doesn't work with the camel-hump format, so `MySql` seems a good compromise. If you don't have strong feelings/arguments about it, I won't change it.